src/Aspire.Cli/Backchannel/AppHostBackchannel.cs

```diff
@@ -196,13 +196,13 @@ await rpc.InvokeWithCancellationAsync(
             cancellationToken).ConfigureAwait(false);
     }
 
-    public async IAsyncEnumerable<CommandOutput> ExecAsync([EnumeratorCancellation] CancellationToken cancellationToken)
+    public async IAsyncEnumerable<BackchannelCommandOutput> ExecAsync([EnumeratorCancellation] CancellationToken cancellationToken)
```

Comment:
This will break compat.

Reply:
With what? This is an internal interface right? With older CLI instances?

Reply:
This _can_ break compat, but for sure when updating the CLI nothing will be broken. I can revert naming change here - the idea was to distinguish an internal backchannel communication interface and the exposed type for getting logs via `ContainerExecService`

Reply:
i reverted the naming change.